PSReadLine.build.ps1

```diff
@@ -342,7 +344,7 @@ task LayoutModule BuildMainModule, BuildMamlHelp, {
 
     # Copy module manifest, but fix the version to match what we've specified in the binary module.
     $version = (Get-ChildItem -Path $targetDir/Microsoft.PowerShell.PSReadLine2.dll).VersionInfo.FileVersion
-    $moduleManifestContent = Get-Content -Path 'PSReadLine/PSReadLine.psd1' -Raw
+    $moduleManifestContent = (Get-Content -Path 'PSReadLine/PSReadLine.psd1' -Raw).Replace("`n","`r`n")
```

Comment:
Since the replace happens in 2 places, you should add a function.

Reply:
Will change to a function